main/res/layout/cachedetail_description_page.xml

```diff
@@ -109,8 +109,7 @@
                 android:drawablePadding="3dip"
                 android:text="@string/cache_menu_spoilers"
                 android:textSize="@dimen/textSize_detailsPrimary"
-                android:textColor="@color/colorText"
```

Comment:
Did you remove the `colorText` intentionally?

Reply:
No, was not on purpose.